.github/CODEOWNERS

```diff
@@ -462,22 +461,29 @@ static/app/components/events/eventStatisticalDetector/                    @getse
+/tests/sentry/api/endpoints/test_organization_metric*                            @getsentry/telemetry-experience
 /src/sentry/api/endpoints/organization_sessions.py                               @getsentry/telemetry-experience
-/src/sentry/api/endpoints/project_dynamic_sampling.py                            @getsentry/telemetry-experience
+/tests/snuba/api/endpoints/test_organization_sessions.py                         @getsentry/telemetry-experience
+/src/sentry/api/endpoints/projects_metrics.py                                    @getsentry/telemetry-experience
+/tests/sentry/api/endpoints/test_projects_metrics_visibility.py                  @getsentry/telemetry-experience
+/src/sentry/api/endpoints/organization_onboarding*                               @getsentry/telemetry-experience
+/tests/sentry/api/endpoints/test_organization_onboarding*                        @getsentry/telemetry-experience
 /src/sentry/dynamic_sampling/                                                    @getsentry/telemetry-experience
+/tests/sentry/dynamic_sampling/                                                  @getsentry/telemetry-experience
 /src/sentry/release_health/metrics_sessions_v2.py                                @getsentry/telemetry-experience
+/tests/sentry/release_health/test_metrics_sessions_v2.py                         @getsentry/telemetry-experience
 /src/sentry/utils/platform_categories.py                                         @getsentry/telemetry-experience
-/src/sentry/api/endpoints/organization_onboarding*                               @getsentry/telemetry-experience
-/tests/sentry/api/endpoints/test_organization_metric_data.py                     @getsentry/telemetry-experience
-/tests/sentry/api/endpoints/test_organization_metric_details.py                  @getsentry/telemetry-experience
-/tests/sentry/api/endpoints/test_organization_metric_tag_details.py              @getsentry/telemetry-experience
-/tests/sentry/api/endpoints/test_organization_metric_tags.py                     @getsentry/telemetry-experience
-/tests/sentry/api/endpoints/test_organization_metrics.py                         @getsentry/telemetry-experience
-/tests/sentry/api/endpoints/test_project_dynamic_sampling.py                     @getsentry/telemetry-experience
-/tests/sentry/api/endpoints/test_organization_dynamic_sampling_sdk_versions.py   @getsentry/telemetry-experience
+/src/sentry/sentry_metrics/querying/                                             @getsentry/telemetry-experience
+/tests/sentry/sentry_metrics/querying/                                           @getsentry/telemetry-experience
+/src/sentry/sentry_metrics/visibility/                                           @getsentry/telemetry-experience
+/tests/sentry/sentry_metrics/visibility/                                         @getsentry/telemetry-experience
+/src/sentry/snuba/metrics/                                                       @getsentry/telemetry-experience
```

Comment:
nit: cant we just target the entire `/src/sentry/sentry_metrics/`

Reply:
We can't since it's snuba that owns parts of that.